src/components/ConversationItem/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { View, TouchableOpacity, Image, Text } from 'react-native';
+import { func } from 'prop-types';
+
+import Avatar from 'components/common/Avatar';
+import { CONVERSATION_SHAPE } from 'constants/shapes';
+import common from 'constants/commonStyles';
+import styles from './styles';
+
+const ConversationItem = ({ conversation, onSelect }) => {
+  const { lastMessage, topicIcon, user } = conversation;
+  const { avatar, fullName } = user;
```

Comment:
Just as a comment, you can also do : 
```suggestion
  const { lastMessage, topicIcon, user: { avatar, fullName} } = conversation;
```